src/OpenTelemetry.Contrib.Extensions.AWSXRay/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
   supporting ResourceBuilderExtensions extension, and migrate all detectors to
   implement OpenTelemetry.Resources.IResourceDetector
   ([#875](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/875))
+* Drop support for `AWSLambdaResourceDetector`.
+  Resources are detected by `OpenTelemetry.Instrumentation.AWSLambda` package
```

Comment:
I guess we could be more clear in the message. Does this make sense?
```suggestion
  AWSLambda Resources are detected by `OpenTelemetry.Instrumentation.AWSLambda` package
```

Reply:
Fixed: 211652a50a59a74350d2d01a61e3b57a39dbb705